src/test/java/com/google/javascript/gents/multiTests/type_rewrite/import.ts

```diff
@@ -1,9 +1,9 @@
 import E from 'goog:keep.E';
 
 import * as A from './imported_module';
-import {A as X} from './imported_module';
+import X from './imported_module';
```

Comment:
We don't support the combined syntax, but I suspect that this still works

(e.g. `import defaultExport, {namedExports} from './file'`)